### PR TITLE
(0.5.6 Device Drivers) unix -> Unix & by a comma? -> by a comma.

### PR DESCRIPTION
**Pull request by alan23273850**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 all: lkmpg.tex
-	pdflatex -shell-escap lkmpg.tex
+	pdflatex -shell-escape lkmpg.tex
```
Comment (jserv) [inline Makefile:2]: Don't touch `Makefile`. Minimize the functional changes.

**Review by jserv — CHANGES_REQUESTED** [review]
The subject was misleading. In fact, the change was not meant to be eliminate the typo.

File: Makefile (hunk 1 above)
Comment (alan23273850) [reply Makefile]: I've already discarded this change!

**Comment by alan23273850** [comment]
> The subject was misleading. In fact, the change was not meant to be eliminate the typo.

OK, I'll be aware of it in the future.

**Comment by jserv** [comment]
Thank @alan23273850 for contributing!

**Pull request merged**